### PR TITLE
Scenario format fixes

### PR DESCRIPTION
**Pull request by hvub**
This PR make a number of small fixes regarding format and style of TCK scenario so that all scenario will pass the upcoming scenario format checker.

This includes 

* minor fixes in the grammar
* adding the @skipGrammarCheck tag to scenario that deliberately violate the grammar
* adding missing scenario numbering
* adjusting of feature names (no camel casing)
* escaping of a variable name where it happens to be a reserved word

### CODE REVIEW

File: tck/features/expressions/list/List11.feature [hunk 1]
```diff
@@ -103,10 +103,10 @@ Feature: List11 - Create a list from a range
     When executing query:
       """
       WITH 0 AS start, [1, 2, 500, 1000, 1500] AS endList, [-1000, -3, -2, -1, 1, 2, 3, 1000] AS stepList
-      UNWIND endList AS end
+      UNWIND endList AS `end`
```
Comment (DvirDukhan) [inline tck/features/expressions/list/List11.feature:106]: I suggest renaming `end` completely

File: tck/index.adoc [hunk 2]
```diff
@@ -14,215 +14,3483 @@ There is also an `uncategorized` and `precategorized` directory containing uncat
 === Call
 
 * Call1 - Basic procedure calling
+
```
Comment (DvirDukhan) [inline tck/index.adoc:17]: what is the reason for the empty lines?

**Review by DvirDukhan — COMMENTED** [review]
looks good
small comments

File: tck/index.adoc (hunk 2 above)
Comment (hvub) [reply tck/index.adoc:17]: That is indeed weird. Good catch. Needs a fix in the generator. I will have look.

Comment (hvub) [reply tck/index.adoc]: fixed

File: tck/features/expressions/list/List11.feature (hunk 1 above)
Comment (hvub) [reply tck/features/expressions/list/List11.feature]: Fixed. Renamed to `stop`.

**Pull request merged**